### DISCUSSION TITLE
how to change navigation bar from the left side to top

### DISCUSSION THREAD

**Discussion by CheapVibe** [netbox-community/netbox#13032] [Q&A]
noticed on some [YouTube videos](https://www.youtube.com/watch?v=O5kayrkuC1E&ab_channel=art) guides that the main menu is on the top
didn't find any reference for how to change the main menu from the default left position to top

would glad for any help

using premier NetBox version v3.5.4

**Comment by abhi1693** [comment]
The navigation which is on top is in NetBox 2.x and it was moved to the side in 3.x.

**Reply by CheapVibe** [reply]
thanks
I find the top bar much more effective for me. any way moving it to top side?
